packages/sonarwhal/src/lib/utils/content-type.ts

```diff
@@ -382,11 +382,38 @@ const isTextMediaType = (mediaType: string): boolean => {
     return false;
 };
 
+const getType = (mediaType: string) => {
+    if (mediaType.startsWith('image')) {
+        return 'image';
+    }
+
+    if (mediaType.startsWith('font')) {
```

Comment:
There is also `application/vnd.ms-fontobject` for EOT fonts.

Reply:
ok, I will add it

Reply:
@sarvaje Can you add this? Thanks!